packages/cozy-harvest-lib/src/models/getRelatedAppsSlugs.js

```diff
@@ -21,15 +24,21 @@ const relatedAppsConfiguration = [
   },
   {
     slug: 'drive',
+    priority: 0,
     predicate: ({ trigger }) => {
       return !!get(trigger, 'message.folder_to_save')
     }
   }
 ]
 
-const getRelatedAppsSlugs = ({ konnectorManifest, trigger }) =>
-  relatedAppsConfiguration
-    .filter(app => app.predicate({ konnectorManifest, trigger }))
-    .map(({ slug }) => slug)
+const getRelatedAppsSlugs = ({ konnectorManifest, trigger }) => {
+  const matchingApps = relatedAppsConfiguration.filter(app =>
+    app.predicate({ konnectorManifest, trigger })
+  )
+
+  return sortBy(matchingApps, ({ priority }) => -priority).map(
+    ({ slug }) => slug
+  )
```

Comment:
Nit: I wonder if this function could let the caller extract the slugs ? I think it would be a bit mor flexible. Anyway this would easily be refactorable.

Reply:
Yeah I changed it because the `predicate` and `priority` don't have a reason to leak outside of this file, but if there are other things we need to expose it would be easy to change :)